### ISSUE TITLE
no settings

### ISSUE THREAD

**Issue by frequency1** [Hopebaytech/redmine_issue_reminder#16]
I have installed plugin .. no errors but it does not show any settings query drop down on right side is always blank
![issue_reminder](https://cloud.githubusercontent.com/assets/11255006/6441228/e5a6f832-c0f6-11e4-8440-e44df38af48d.PNG)

**Comment by Androc** [comment]
Hello,

did you have saved query ? Go to "Issues", do the search you want, with the options you want, then save it, after that you will be able to use it in plugin.

**Comment by frequency1** [comment]
I have existing issues with status set as new. I opened those issues in edit mode, changed status to in progress , saved but still same as shown in screen shot.

**Comment by Androc** [comment]
I think you do not understand what this plugin does.

According to the README :
> Plugin provides an easy to use interface to set up automatic email reminder to every project. Every reminder uses a custom query with all their filter options to select issues and performs periodical email transmission on a role basis.

For example, if you want to regularly inform your developpers about "opened issues that were not updated since 5 days".
You need to create a custom query that will search for these issues and configure the plugin to remind them every 2 days (for example) to your developpers.

If you do not know how to create custom queries, the README will help you : https://github.com/Hopebaytech/redmine_issue_reminder#how-can-i-customize-the-queries

Basically you have to go to http://**youserver**/projects/**oneofyourprojects**/queries/new, fill the form, then save it.

**Comment by frequency1** [comment]
Thanks a ton ... i am able to create issue reminders now. But E-Mail still doesn't go. I have checked mail relay settings and it is correct as i am able to receive default notifications from redmine builtin features. I executed below to check ... but no e-mails
[redmine]# rake --trace reminder:exec RAILS_ENV=production
** Invoke reminder:exec (first_time)
** Invoke environment (first_time)
** Execute environment
** Execute reminder:exec
[redmine]# rake --trace reminder:exec[test]
** Invoke reminder:exec (first_time)
** Invoke environment (first_time)
** Execute environment
DEPRECATION WARNING: ActionDispatch::Assertions::SelectorAssertions has been extracted to the rails-dom-testing gem. (called from <top (required)> at /usr/local/rvm/gems/ruby-1.9.3-p551/bundler/gems/prototype-rails-bfec47366d48/lib/prototype-rails/selector_assertions.rb:3)
** Execute reminder:exec
XXX@XXX.com
YYY@YYY.com

**Comment by Androc** [comment]
If you have xxx@xxx.com and yyy@yyy.com it is on the good way.

Are you able to send test email `rake reminder:exec[test] RAILS_ENV=production` ?

If yes, your plugin probably works but you missed an important information from README (you are not the first one : #3) : https://github.com/Hopebaytech/redmine_issue_reminder#testing-redminder-mail

The plugin stores the last execution date and does not send email if last execution date is under the frequency of the issue reminder's configuration.

**Comment by amilcarb** [comment]
Same issue :-(

Here is the log:

Started POST "/issue_reminders" for 78.192.27.16 at 2015-03-20 19:10:25 +0100
Processing by IssueRemindersController#create as JS
  Parameters: {"utf8"=>"?", "reminder"=>{"project_id"=>"3", "query_id"=>"3", "interval"=>"daily"}, "interval_value"=>"0", "manager"=>"3", "commit"=>"Add"}
  Current user: abatista (id=8)
Completed 500 Internal Server Error in 29ms

NameError (undefined local variable or method `parameters' for #<IssueRemindersController:0x0000000513fce0>):
  plugins/redmine_issue_reminder/app/controllers/issue_reminders_controller.rb:23:in `create'
  actionpack (4.2.0) lib/action_controller/metal/implicit_render.rb:4:in `send_action'
  actionpack (4.2.0) lib/abstract_controller/base.rb:198:in `process_action'
  actionpack (4.2.0) lib/action_controller/metal/rendering.rb:10:in `process_action'
  actionpack (4.2.0) lib/abstract_controller/callbacks.rb:20:in `block in process_action'
  activesupport (4.2.0) lib/active_support/callbacks.rb:117:in `call'
  activesupport (4.2.0) lib/active_support/callbacks.rb:117:in `call'
  activesupport (4.2.0) lib/active_support/callbacks.rb:151:in `block in halting_and_conditional'
  activesupport (4.2.0) lib/active_support/callbacks.rb:169:in `call'
  activesupport (4.2.0) lib/active_support/callbacks.rb:169:in `block in halting'
  activesupport (4.2.0) lib/active_support/callbacks.rb:169:in `call'
  activesupport (4.2.0) lib/active_support/callbacks.rb:169:in `block in halting'
  activesupport (4.2.0) lib/active_support/callbacks.rb:169:in `call'

**Comment by jcppkkk** [comment]
@amilcarb it's actually different issue. however the fix still comes f9bf7059366d2d320e59ece3e2e843108ac60363. Please pull the changes and try again :+1:

**Comment by Androc** [comment]
By the way, I think this issue can be closed
